biocrnpyler/utils.py

```diff
@@ -127,6 +129,9 @@ def member_dictionary_search(member,dictionary):
                         warn(f"dictionary search output was {out_value} but now it will be {cur_value}")
                     out_value = cur_value
 
+            if 'reaction' in dictionary and out_value is None:
+                out_value = dictionary['reaction']
```

Comment:
i think i disagree with having 'reaction' be a special word for this search function. It is supposed to be a generic dictionary search function and not specifically for use with plotting

Reply:
ok, we can remove that check here and just do it manually in plotting

Reply:
resolved on next commit.